web/app/scripts/src/providers/project-config.es.js

```diff
@@ -0,0 +1,22 @@
+const ProjectConfig = function () {
+    let settings = {
+        apiRoot: 'localhost:8000'
+    };
+
+    return {
+        $get: function () {
+            return {
+                getApiRoot: () => settings.apiRoot
+            }
+        },
+        setApiRoot: (value) => {
+            settings.apiRoot = value;
+        }
+    };
+}
+
+export const provider = [
```

Comment:
Ah, I see what you mean.

What about registering a new module in this file, and exporting that module's name?
```js
import ProjectConfigModule from 'project-config.js';

angular.module('incuna-surveys', [
    ProjectConfigModule,
    // ...
]);
```

Then `project-config` can deal with itself. We've done that before so that outside code doesn't have to manage dependencies of separate files. It may feel pointless to have one module per Angular _thing_, but it solves the dependencies-pattern problem and means each file can be read and understood independently.